tests/trustyai/lm_eval/utils.py

```diff
@@ -0,0 +1,18 @@
+from kubernetes.dynamic import DynamicClient
+from ocp_resources.lm_eval_job import LMEvalJob
+from ocp_resources.pod import Pod
+
+from timeout_sampler import retry
+
+
+def verify_lmevaljob_running(client: DynamicClient, lmevaljob: LMEvalJob) -> None:
+    # Verifies that the LMEvalJob Pod gets to Running and stays either Running or Succeeded for at least 2 minutes
```

Comment:
I don't see where you check here that the status is what you expect for 2 minutes.